tests/models/test_models_unet_2d.py

```diff
@@ -151,7 +148,7 @@ def test_from_pretrained_accelerate_wont_change_results(self):
         gc.collect()
 
         model_normal_load, _ = UNet2DModel.from_pretrained(
-            "fusing/unet-ldm-dummy-update", output_loading_info=True, device_map="auto"
+            "fusing/unet-ldm-dummy-update", output_loading_info=True, fast_init=False
```

Comment:
@pcuenca @patrickvonplaten adapted the tests a bit.

now passing `fast_init=False` here which will load the model without `accelerate`.